neo/test/controllers/ApplicationTest.scala

```diff
@@ -0,0 +1,36 @@
+package controllers
+
+import java.io.File
+
+import org.specs2.mutable.Specification
+import play.api.libs.json.JsArray
+
+/**
+  * Created by zongh on 10/26/2016.
+  */
+class ApplicationTest extends Specification {
+
+  "application" should {
+
+    val cities = Application.loadCity(new File("public/data/city.sample.json"))
```

Comment:
can you move this `city.sample.json` into `test/resource/data` folder and update the correspoding code path?